package.json

```diff
@@ -67,5 +67,9 @@
     "shelljs",
     "xcode",
     "xml-escape"
-  ]
+  ],
+  "nyc": {
```

Comment:
If I would do `npm install` it writes the following change to `package.json`:
```diff
index 7b21884a..ee6b352a 100644
--- a/package.json
+++ b/package.json
@@ -69,7 +69,12 @@
     "xml-escape"
   ],
   "nyc": {
-    "include": ["bin/templates/scripts/**"],
-    "reporter": ["lcov", "text"]
+    "include": [
+      "bin/templates/scripts/**"
+    ],
+    "reporter": [
+      "lcov",
+      "text"
+    ]
   }
 }
```

I would favor having the "nyc" entry committed this way.

Reply:
Are you referring to the formatting, each item on its own line? I can make this change.

Reply:
Yes. I have a few other general remarks coming on this nice PR.